plugins/dbviewer/frontend/public/javascripts/countly.models.js

```diff
@@ -6,22 +6,31 @@
 		_document = {};
 
     //Public Methods
-    countlyDBviewer.initialize = function () {
+    countlyDBviewer.initialize = function (app_id) {
+    	var data = { api_key: countlyGlobal['member'].api_key, app_id:app_id };
+		// is app_id provided?
+		if (data.app_id && data.app_id == "all") delete data.app_id;
```

Comment:
Still confused about this
1) deleting is costly operation, adding something just to delete it seems bad in general
2) So if app_id is not provided, we have data.app_id = undefined.  I know it won't do anything bad, but the logic breaks me, sorry. :D 

can we just do 

```
var data = { api_key: countlyGlobal['member'].api_key};
if(data.app_id && data.app_id !== "all"){
    data.app_id = app_id;
}
```